packages/create/index.js

```diff
@@ -8,12 +9,26 @@ import path from 'path';
 import prompts from 'prompts';
 
 const templates = [
-  'default',
-  'avatar-with-background',
-  'google-cloud-run',
-  'google-cloud-run-parallelized',
-  'stitching-videos',
-  'youtube-shorts',
+  {
+    value: 'saas-template',
+    title: 'Revideo with Next.js (recommended)',
+    description: 'An example project to get started with Revideo and Next.js.',
+  },
+  {
+    value: 'default',
+    title: 'Minial, standalone Revideo project',
+    description: 'A minimal example to get started with Revideo.',
+  },
+  {
+    value: 'google-cloud-run',
+    title: 'Google Cloud Run',
+    description: 'Google Cloud Run deployment.',
```

Comment:
I would make this a little more descriptive, to make clear that this is about deploying a rendering service. Maybe something like "Render a Revideo Project on Google Cloud Run" or "Deploy a Revideo Project on Google Cloud Run for Rendering"